components/TopCharts/TopCharts.jsx

```diff
@@ -47,8 +47,8 @@ const TopCharts = ({ topCharts }) => {
                     <Image
                       src={chart?.images.coverart}
                       alt={chart?.title}
-                      width={16}
-                      height={16}
+                      width='100%'
```

Comment:
Instead of width & height 100% we can use the layout="fill" and set the parent container "relative"